kwanj-k/storemanager-API
# #161210690 Add a create store endpoint

### What does this PR do?
Adds an endpoint user to add a store to the manager 

#### Description of Task to be Completed ?
-- Model a store data class
-- Make an add store endpoint
-- Validate user inputs

### How should this be manually tested?
* After cloning the repo,git pull the branch ft-new-store- #161210690
* Install a virtual enviroment and activate it [Refer here](https://docs.python-guide.org/dev/virtualenvs/)
* Pip install the the requirements.txt
* Run the following:
  ``` export FLASK_APP='run.py' ```
  ``` export APP_SETTINGS='development' ```
   ``` pytest --cov-report term-missing --cov=app ```

 All related products tests should pass

### What are the relevant Pivotal Tracker stories?
   #161210690
